src/app.ts

```diff
@@ -0,0 +1,7 @@
+import config from "./config";
+import { Product, User } from "./models";
+
+console.log((<any>config).name);
```

Comment:
you can remove this and use just `console.log(config.name)` following the approach I described below in comment for `tsconfig`